doc/source/how-to/manage-protobuf-definitions.rst

```diff
@@ -0,0 +1,163 @@
+
+.. code-block:: yml
+
+    name: Nightly dev release
+  
+    on:
+      schedule: # UTC at 0300
+        - cron: "0 5 * * *"
+      workflow_dispatch:
+          
+    env:
+      MAIN_PYTHON_VERSION: '3.10'
+    
+    jobs:
+    nightly-release:
+        name: Nightly release package
+        if: github.ref == 'refs/heads/main'
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v3
+    
+        - name: Append date to version tags
+            run: |
+            PACKAGE_DATE=$(date +'%Y%m%d%H%M')
+            sed -i "1s/.$/$PACKAGE_DATE/" ansys/api/<api-name>/VERSION
+        - name: Set up Python
+            uses: actions/setup-python@v4
+            with:
+            python-version: ${{ env.MAIN_PYTHON_VERSION }}
+    
+        - name: Install build requirements
+            run: |
+            pip install -U pip
+            pip install build
+        - name: Build
+            run: python -m build
+    
+        - name: Install
+            run: pip install dist/*.whl
+    
+        - name: Test import
+            run: |
+            mkdir tmp
+            cd tmp
+            python -c "import ansys.api.<api-name>.v0; print('Successfully imported ansys.api.<api-name>.v0')"
+            python -c "from ansys.api.<api-name> import __version__; print(__version__)"
+        - name: Upload to PyPI
+            run: |
+            pip install twine
+            twine upload --skip-existing ./**/*.whl
+            twine upload --skip-existing ./**/*.tar.gz
+            env:
+            TWINE_USERNAME: PAT
+            TWINE_PASSWORD: ${{ secrets.PYANSYS_PyPI_PAT }} 
+            TWINE_REPOSITORY_URL: https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/PyPI/upload
+    
+        - name: Upload packages
+            uses: actions/upload-artifact@v3
+            with:
+            name: ansys-api-<api-name>-packages
+            path: dist/
+            retention-days: 7
+
+PyPI packages follow semantic versioning while gRPC Protobuf API versions typically follow a simplified v*
+versioning pattern. It is not expected to synchronize PyPI package version with Protobuf API version.
+There is no methodology to correlate PyPI package version with exposed gRPC API versions included within
+the package.
+
+
+Consuming the API package within Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the API package has been published to PyPI, a reference can be included within
+the client library build dependencies.
+
+Example poetry configuration:
+
+.. code-block:: toml
+
+    [tool.poetry.dependencies]
+    python = ">=3.7,<4.0"
+    ansys-api-<api-name> = {version = "==*.*.*", source = "PyPI"}
+
+The stub imports follow a standard pattern. For each API service, there is a ***_pb2
+module which defines all of the messages defined within that specific service file and
+a ***_pb2_grpc module that defines a Stub class that encapsulates all service methods.
```

Comment:
Not sure why this format is used:  ***_pb2_grpc.

Reply:
Adapted!